lib/tasks/assets.rake

```diff
@@ -36,6 +34,16 @@ task 'assets:precompile:before' do
 
   require 'sprockets'
   require 'digest/sha1'
+
+  if ENV['EMBER_CLI_PROD_ASSETS']
+    # Remove the assets that Ember CLI will handle for us
+    Rails.configuration.assets.precompile -= %w(application.js admin.js ember_jquery.js pretty-text-bundle.js start-discourse.js markdown-it-bundle.js)
+    Rails.configuration.assets.precompile.reject! do |asset|
+      asset.is_a?(String) &&
+        (%w(application.js admin.js ember_jquery.js pretty-text-bundle.js start-discourse.js vendor.js).include?(asset) ||
```

Comment:
Are the two lists necessary or can they be merged? (into the `reject!` block?)

Reply:
Good catch. That previous line was my failed attempt to remove them. Only `reject!` is required and I've fixed it.